### PR TITLE
Fix for the spa containers : Accordion, Tabs , Carousel when inserting components

### PR DESCRIPTION
**Pull request by niekraaijmakers**
## Description

The fix is to simply have the 'new' node present under the apps, this is all that's needed for the spa editor to work.

## Related Issue

https://github.com/adobe/aem-project-archetype/issues/700

## Motivation and Context

It fixes the SPA containers in edit mode: Accordion, Tabs and Carousel.

## How Has This Been Tested?

Localhost testing.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
     <groupId>com.adobe.aem</groupId>
     <artifactId>aem-project-archetype</artifactId>
-    <version>27-SNAPSHOT</version>
+    <version>28-SNAPSHOT</version>
```
Comment (habansal) [inline pom.xml:22]: why is this needed ?

**Review by habansal — COMMENTED** [review]
Questions:
* Do we need this `new` node for only the SPA projects or for the non-spa projects as well?
* If only needed for SPA projects -
  -  why ? what's the difference ?
  -  how are we ensuring that it only applies to SPA projects ?
* If needed for both, should this `new` node come from the Core components instead ?

**Review by vladbailescu — APPROVED** [review]
